root/static/scripts/edit/URLCleanup.js

```diff
@@ -4573,19 +4574,24 @@ const CLEANUPS: CleanupEntries = {
               result: prefix === 'artist' || prefix === 'org',
               target: ERROR_TARGETS.ENTITY,
             };
-          case LINK_TYPES.vgmdb.release:
+          case LINK_TYPES.vgmdb.event:
             return {
-              result: prefix === 'album',
+              result: prefix === 'event',
```

Comment:
Fixed the order to be alphabetical per entity type, maybe I should have made this a second commit, but hopefully it's not too annoying :)

Reply:
Hopefully it's not too annoying to split this commit either.

Reply:
Fiiiiiine, I'll reorder separately :)